src/java/org/apache/cassandra/db/compaction/unified/Controller.java

```diff
@@ -69,6 +70,7 @@ public abstract class Controller
     private static final ConcurrentMap<TableMetadata, Controller.Metrics> allMetrics = new ConcurrentHashMap<>();
 
     static final String PREFIX = "unified_compaction.";
+    static final String DSE_PREFIX = "dse." + PREFIX;
```

Comment:
Nit: I don't think there's a need to specifically prefix this one. Let it be just `unified_compaction.num_shards`.

Reply:
I wanted to support all the occurrences of this setting. I see them being set in at least 2 repositories I'm betting that out there in the wild we have some occurrences set manually in k8s. I realize it doesn't fit ds-trunk very well, but hopefully we could remove that soon (famous last words).

Reply:
I reverted the prefix.